#ifndef __TOPIC_H
#define __TOPIC_H

// Topic images
constexpr const char* const TOPIC_RGB_IMAGE = "/camera/rgb/image_raw";
constexpr const char* const TOPIC_DEPTH_IMAGE = " /camera/depth/image_raw";

// Topic hal
constexpr const char* const TOPIC_BUTTON_1 = "/bt1_status";
constexpr const char* const TOPIC_BUTTON_2 = "/bt2_status";
constexpr const char* const TOPIC_BUTTON_3 = "/bt3_status";
constexpr const char* const TOPIC_BUTTON_4 = "/bt4_status";
constexpr const char* const TOPIC_SS_STATUS = "/ss_status";
constexpr const char* const TOPIC_LED_STATUS = "/led_status";
constexpr const char* const TOPIC_LCD_PRINT = "/lcd_print";

// Topic car_controller
constexpr const char* const TOPIC_SET_SPEED = "/set_speed_car_api";
constexpr const char* const TOPIC_SET_STEER = "/set_steer_car_api";
constexpr const char* const TOPIC_SET_POS_CAMERA = "/set_pos_camera_api";

// Topic MPU







#endif